lib/betagouvbot/notification_request.rb

```diff
@@ -15,11 +15,13 @@ def call(members, date)
       private
 
       def warnings(members, date)
-        NotificationSchedule.(members, horizons, date)
+        NotificationSchedule.(members, end_dates(date))
       end
 
-      def horizons
-        NotificationRule.horizons
+      def end_dates(date)
```

Comment:
Might as well move `end_dates` to `NotificationRule` (or `NotificationSchedule`) then?